File: Java_Weather_API_Library/src/main/java/command/ConfigCommand.java
package command;

import manager.ConfigJson;
import manager.DeserializedJson;
import manager.JsonManager;

public class ConfigCommand implements CommandBase {

    @Override
    public void runCommand(String name) {
        if (name.equalsIgnoreCase("get_config")) {
            JsonManager configManager = new JsonManager(JsonManager.FileType.CONFIG);
            ConfigJson deserializedJson = (ConfigJson) configManager.getDeserializedJson();
            System.out.print("生データ：" + configManager.getRawElement());
        }
    }

}

File: C-Sharp_Weather_API_Library/ConfigCommand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace C_Sharp_Weather_API_Library
{
    internal class ConfigCommand : CommandBase
    {
        public void runCommand(string command)
        {
            if (command.Equals("get_config"))
            {
                ConfigJson configJson = ConfigManager.GetConfig();
                Console.WriteLine("生データ：" + ConfigManager.GetRawJson());
            }
        }
    }
}
